google/cloud/pubsub_v1/subscriber/message.py

```diff
@@ -429,12 +467,17 @@ def nack(self) -> None:
         may take place immediately or after a delay, and may arrive at this subscriber
         or another.
         """
+        if self.opentelemetry_data:
+            self.opentelemetry_data.add_subscribe_span_event("nack start")
```

Comment:
Same comment as ack, event should be added when the RPC is invoked

Reply:
Done. Response https://github.com/googleapis/python-pubsub/pull/1252#discussion_r1769706317